OrangeHRMTests/PageObjects/LoginPage.cs

```diff
@@ -0,0 +1,45 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.Drivers;
+using OrangeHRMTests.Common.WebElements;
+using OrangeHRMTests.Data;
+
+namespace OrangeHRMTests.PageObjects
+{
+    public class LoginPage : BasePage
+    {
+        private MyWebElement UserNameField = new MyWebElement(By.XPath("//*[contains(@class, 'form-row')]//*[contains(@name, 'username')]"));
+        private MyWebElement PassWordField = new MyWebElement(By.XPath("//*[contains(@class, 'form-row')]//*[contains(@name, 'password')]"));
+        private MyWebElement InvalidMessageTextElement = new MyWebElement(By.XPath("//*[contains(@class, 'login-error')]//*[contains(@class, 'alert-content-text')]"));
+        private MyWebElement ForgotPasswordButton = new MyWebElement(By.XPath("//*[contains(@class, 'login-forgot')]/p"));
+        private MyWebElement ResetPasswordMessageTextElement = new MyWebElement(By.XPath("//*[contains(@class, 'forgot-password-title')]"));
+
+        public string GetCurrentPageUrl() => WebDriverFactory.Driver.Url;
+
+        public void EnterValidUsername() => UserNameField.SendKeys(TestSettings.Username);
+
+        public void EnterValidPassword() => PassWordField.SendKeys(TestSettings.Password);
+
+        public void ClickLoginButton() => BasePage.ClickSaveButton();
```

Comment:
1.no need to use BasePage., just ClickSaveButton(), please correct on every page
2. For me using locator of save button for login button is not a good idea... because your Save button  is described only by submit property. any button in app can have this property, it's not reliable at all. Better write locator base on text
![image](https://github.com/agrren/ReneiskiDiploma/assets/55654931/af4ee6a9-1ac7-4a31-afd7-f44b750a89ab)

Reply:
Fixed